### PR TITLE
WSJF prioritization model

### PR DESCRIPTION
**Pull request by gabrielcardoso21**

### CODE REVIEW

File: project_wsjf/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,37 @@
+# -*- coding: utf-8 -*-
+# Copyright 2019 Kmee Informática LTDA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    'name': 'Project Wsjf',
+    'summary': """
+        This module implements the Weighted Shortest Job First prioritization model.
```
Comment (mileo) [inline project_wsjf/__manifest__.py:8]: Can u please move the summary to the README.rst and fill the other questions like, Installation, Configure, Contributors and etc.. ?

And check the travis erros.

**Review by mileo — CHANGES_REQUESTED** [review]

**Comment by mileo** [comment]
More info about https://www.scaledagileframework.com/wsjf/

File: project_wsjf/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+    """,
+    'version': '12.0.1.0.0',
+    'license': 'AGPL-3',
+    'author': 'Kmee Informática LTDA,Odoo Community Association (OCA)',
+    'website': 'www.kmee.com.br',
```
Comment (mileo) [inline project_wsjf/__manifest__.py]: Change the website to https://github.com/OCA/project/

**Review by mileo — CHANGES_REQUESTED** [review]

File: project_wsjf/__manifest__.py (hunk 1 above)
Comment (dreispt) [reply project_wsjf/__manifest__.py:8]: Summay should stay here. Description is to the replaced by the readme.

File: project_wsjf/models/project_task.py [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+class ProjectTask(models.Model):
+    _name = 'project.task'
+    _inherit = [
+        'project.task',
+        'project.wsjf.mixin',
+    ]
+    _order = 'wsjf desc, sequence'
+
+    project_id = fields.Many2one(
+        inverse='_inverse_project_id',
```
Comment (dreispt) [inline project_wsjf/models/project_task.py]: Can you explain this?
AFAIK project_id is not a computed field, why define an Inverse computation function for it?

File: project_wsjf/models/project_project.py [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+class ProjectProject(models.Model):
+    _name = 'project.project'
+    _inherit = [
+        'project.project',
+        'project.wsjf.mixin',
+    ]
+    _order = 'wsjf desc, sequence'
+
+    internal_pressure = fields.Selection(
+        inverse='_inverse_internal_pressure'
```
Comment (dreispt) [inline project_wsjf/models/project_project.py]: I see this is extending the mixin. But it is a not a computed field, why the inverse computation?

File: project_wsjf/README.rst [hunk 5]
```diff
@@ -0,0 +1,91 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
```
Comment (dreispt) [inline project_wsjf/README.rst]: Ot would me nice to use the new readme structure, that automatically generates the top level README:
https://github.com/OCA/maintainer-tools/tree/master/template/module/readme

**Review by dreispt — CHANGES_REQUESTED** [review]
Can you please fix the build?

**Pull request closed without merging**